### ISSUE TITLE
Toasts should not auto hide by default

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#4433]
**Description**
Toasts should remain open by default. There aren't cases as of now where they should hide after a certain amount of time due to accessibility, but if there are they should be set on a case by case basis instead of disappearing after 3 seconds by default. 

**Additional context**
This was brought up in a UX review by @mferhut
